pkg/framework/simulator_test.go

```diff
@@ -190,11 +195,23 @@ func TestPrediction(t *testing.T) {
 
 	// 2. create predictor
 	// - create simple configuration file for scheduler (use the default values or from systemd env file if reasonable)
-	soptions, _ := sapps.NewOptions()
-	ksConfig := new(componentconfig.KubeSchedulerConfiguration)
-	ksConfig, _ = soptions.ApplyDefaults(ksConfig)
-	schedServer, _ := sapps.NewSchedulerServer(ksConfig, "http://localhost:8080")
-	cc, err := New(schedServer,
+	opts, err := kubescheduleroptions.NewOptions()
+	if err != nil {
+		t.Fatalf("unable to create scheduler options: %v", err)
+	}
+
+	// inject scheduler config config
+	opts.ComponentConfig = kubeschedulerconfig.KubeSchedulerConfiguration{
+		SchedulerName: "default-scheduler",
```

Comment:
Can you use `k8s.io/api/core/v1.DefaultSchedulerName` instead?

Reply:
changed  "default-scheduler" to "k8s.io/api/core/v1".DefaultSchedulerName